var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -59,8 +59,8 @@ class NetcdfC(CMakePackage, AutotoolsPackage):
         patch("4.8.1-win-hdf5-with-zlib.patch", when="@4.8.1: platform=windows")
 
         # TODO: fetch from the upstream repo once https://github.com/Unidata/netcdf-c/pull/2595
-        #  is accepted:
-        patch("netcdfc-mpi-win-support.patch", when="platform=windows")
+        #  is accepted and the HDF5 interface patch is added
```

Comment:
I'm reading this todo comment and getting confused:

* It sounds like https://github.com/Unidata/netcdf-c/pull/2595 is no longer sufficient
* "and the HDF5 interface patch is added" makes it sound like some other PR is required, but are the changes here not part of some PR to netcdf-c yet?

Taken together, I think the comment should be updated to something like:

```
# https://github.com/Unidata/netcdf-c/pull/2595 contains some of the changes in this patch but is not
# sufficient. There is no upstream PR that includes all these changes.
```

Reply:
I upstreamed the MPI changes when I made them months ago, I just haven't had a chance to upstream the HDF5 related changes yet so there is no PR that encompasses the entire patch. 

Will update the comment.